### ISSUE TITLE
Why not release on mac store

### ISSUE THREAD

**Issue by tami5** [sbarex/QLMarkdown#65]
Hi there, I wonder why this is not release on the mac store, I was using https://apps.apple.com/us/app/previewmarkdown/id1492280469?mt=12 and it has just 5% of what QLMarkdown provide. 

Please consider providing an easy access for mac user in exchange for some financial tokens like 2.99$. 

Best of luck

**Comment by hazarek** [comment]
@tami5 Homebrew package manager is easier and more practical than Appstore.

**Comment by elsiehupp** [comment]
The Mac App Store doesn't have to be mutually exclusive with Homebrew, and the "notarization" process for the Mac App Store would make Homebrew installs work better by not requiring the user to launch the application in order to manually dismiss the code-signing warning. The main advantage with the Mac App Store would be increased visibility and easier installation for end users who prefer it.

**Comment by elsiehupp** [comment]
tl;dr Why not do ***both*** Homebrew and the Mac App Store?

**Comment by hazarek** [comment]
> tl;dr Why not do _**both**_ Homebrew and the Mac App Store?

Because Apple is an enemy of open source software. The App Store is full of garbage. I've never needed the Apple store.

**Comment by elsiehupp** [comment]
> > tl;dr Why not do _**both**_ Homebrew and the Mac App Store?
> 
> Because Apple is an enemy of open source software. The App Store is full of garbage.

That would kind of be an answer if you were the maintainer of this, but you aren’t, so it just comes across as kind of bossy?

> I've never needed the Apple store.

This is a non-sequitur, given that the benefits of distributing this on the Mac App Store are ***for people other than you***.

Your response is just fundamentally unserious. You didn’t open this issue, so why not just unsubscribe from it? You clearly have nothing productive to add to the conversation.

**Comment by hazarek** [comment]
>That would kind of be an answer if you were the maintainer of this, but you aren’t, so it just comes across as kind of bossy?
I'm just stating my opinion. within the scope of freedom of expression. I am originator and contributor(theme, logo) of this FOSS project. So who are you?

**Comment by elsiehupp** [comment]
> That would kind of be an answer if you were the maintainer of this, but you aren’t, so it just comes across as kind of bossy?

You’re telling other people not to do something, which is bossy. And it’s entirely off-topic for this thread if you don’t feel like doing Mac App Store packaging yourself.

Again, nothing you’ve said here contributes to the conversation in any meaningful way. You are deliberately derailing this thread.

**Comment by hazarek** [comment]
>Again, nothing you’ve said here contributes to the conversation in any meaningful way.

I think it is very MEANINGFUL, I put my political attitude on behalf of open source software.

**Comment by sbarex** [comment]
At the moment this application is not published on the App Store for the following reasons: 

- the developer license has an annual cost (which is why the precompiled version is not digitally signed)
- charging the application involves figuring out how much it costs in taxes and it would probably not be worth it anymore 
- probably I'm using some security exceptions that would not be accepted by the App Store 
- the software is developed as a hobby and would probably need a lot of tweaking

**Comment by abhibeckert** [comment]
> the developer license has an annual cost (which is why the precompiled version is not digitally signed)

There is no annual cost for code signing - that's free. The fee is only for hosting apps in the store.

**Comment by sbarex** [comment]
> > the developer license has an annual cost (which is why the precompiled version is not digitally signed)
> 
> Signing is free and it can be an automated step in your build script or alternatively Xcode provides a nice GUI.
> 
> The annual fee is only for hosting apps in the store.

If I remember correctly, however, the certificate to sign the applications expires in a week and then must be regenerated and the application must be recompiled. Or is it a limitation only for iOS applications?

**Comment by sbarex** [comment]
From today, anyone who wants can contribute with small (or large :smile:) financing by [buy me one (or more) coffees!](https://www.buymeacoffee.com/sbarex)

Thank you all!
